pandas/tests/indexes/test_range.py

```diff
@@ -315,6 +315,16 @@ def test_is_monotonic(self):
         self.assertTrue(index.is_monotonic_increasing)
         self.assertTrue(index.is_monotonic_decreasing)
 
+        index = RangeIndex(2, 1)
+        self.assertTrue(index.is_monotonic)
+        self.assertTrue(index.is_monotonic_increasing)
+        self.assertTrue(index.is_monotonic_decreasing)
+
+        index = RangeIndex(1, 1)
+        self.assertTrue(index.is_monotonic)
+        self.assertTrue(index.is_monotonic_increasing)
+        self.assertTrue(index.is_monotonic_decreasing)
+
```

Comment:
@jorisvandenbossche empty case is coverd by ``(1, 1)``.

```
pd.RangeIndex(1, 1).values
# array([], dtype=int64)
```

Let me issue a separate PR to prohibit empty construction.